### DISCUSSION TITLE
Job

### DISCUSSION THREAD

**Discussion by sachin-mishra13** [community/community#31573] [Other Feature Feedback, Questions, & Ideas]
I'm beginner which language i can choose to learn

**Comment by toufiqul-hasan** [comment]
Python because it's a beginner friendly programming language.

**Reply by anasmak04** [reply]
I think is better to start learn how the computer works. And when you use closer-to-hardware languages like C/C++ you get a better idea of how a computer works when you learn to code.  and you can choose any language to start 
^_^

**Reply by airtower-luna** [reply]
Starting with a language that provides *unchecked* memory access is not a good idea, even experienced programmers frequently create bugs (including many, probably *most* security issues) that way. If you want a safe close-to-hardware language to start with, Rust or Go are good choices.

Otherwise I strongly agree with the Python recommendation. :slightly_smiling_face:

**Comment by melvincwng** [comment]
@sachin-mish Hi Sachin, welcome to GitHub Community 👋

For starters, I would recommend either JavaScript or Python, since they are very beginner friendly languages. Syntax wise they are a lot easier to understand as compared to some of the more verbose languages out there.

Hope this answers your question, and happy learning 😊

**Comment by jfullstackdev** [comment]
check this out, it's my reply on the same topic

https://github.com/community/community/discussions/29780#discussioncomment-3415347
